### PR TITLE
fix(test_optimizer): mismatch of versions on hook

### PR DESCRIPTION
**Pull request by renancaraujo**
## Description

fix(test_optimizer): mismatch of versions on hook

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [x] 🗑️ Chore

### CODE REVIEW

File: bricks/test_optimizer/hooks/pubspec.yaml [hunk 1]
```diff
@@ -8,7 +8,10 @@ dependencies:
   mason: ">=0.1.0-dev.41 <0.1.0"
   path: ^1.8.1
 
+
+# Beware: on hooks, even dev dependencies have to be compatible to all dart versions covered by
```
Comment (alestiago) [inline bricks/test_optimizer/hooks/pubspec.yaml:12]: Is there a way we can automate checking this?

**Review by alestiago — APPROVED** [review]

File: bricks/test_optimizer/hooks/pubspec.yaml (hunk 1 above)
Comment (wolfenrain) [reply bricks/test_optimizer/hooks/pubspec.yaml:12]: E2e testing of the test command

**Review by wolfenrain — APPROVED** [review]

File: bricks/test_optimizer/hooks/pubspec.yaml (hunk 1 above)
Comment (renancaraujo) [reply bricks/test_optimizer/hooks/pubspec.yaml:12]: Added https://github.com/VeryGoodOpenSource/very_good_cli/issues/676 for this

**Comment by MauriMiguez** [comment]
Thanks for taking the time and solving the issue! 🚀

**Pull request merged**